frontend/ui/elements/font_ui_fallbacks.lua

```diff
@@ -56,6 +56,7 @@ Fonts for many languages can be downloaded at:
 https://fonts.google.com/noto
 
 Only fonts named "Noto Sans xyz" or "Noto Sans xyz UI" (regular, not bold nor italic, not Serif) will be available in this menu.]])
+more_info_text = more_info_text .. "\n" .. _("However, bold fonts will be used if their corresponding regular fonts exist.")
```

Comment:
You should probably just include this in the original `more_info_text`

Reply:
I was trying to keep the old translations. If I just change the old `more_info_text`, will the old translations be invalid and the translators not know there's a big chunk of it already translated? 🤔

Reply:
I think translators will get the previous text. @Frenzie will confirm.
Anyway, best to keep it as a single chunk of text (if we had done these patch/concatenations everytime we added stuff, you would have meet more of them in our existing code :)

Reply:
You lost your added `\n` when making it into the single string (which can just be a real newline in this Lua `[[string]]`).

Reply:
It's on purpose :)

Reply:
If the text is identical then it'll already be translated, yes. But the translation history should also perform a similar role. We don't want to give the translators unnecessary work but we don't have to bend over backwards for it. :-)